make/buf/all.mk

```diff
@@ -1,6 +1,5 @@
-# https://github.com/jhump/protoreflect/commits/master 20220416 checked 20220417
```

Comment:
Update to `https://github.com/bufbuild/protocompile/commits/main DATE_OF_LAST_COMMIT checked DATE_WE_LAST_CHECKED_FOR_A_NEW_COMMIT`